translations/stripes-acq-components/en.json

```diff
@@ -90,6 +90,7 @@
   "sources.marc": "MARC",
   "sources.user": "User",
 
+  "resultsCount": "{count, number} {count, plural, one {record found} other {records found}}",
```

Comment:
I guess here you expect `resultsCount` instead of `count` - [line 15](https://github.com/folio-org/stripes-acq-components/pull/594/files#diff-94c8763884ab6d65f85dbe851e3b9cfeff1bcda55664ea9f4a32bcc0fa1a4e97R15)

Reply:
oh, yep, copy paste